client/components/fastlane/html/src/v6-fastlane.js

```diff
@@ -0,0 +1,69 @@
+async function onLoad() {
+  async function getBrowserSafeClientToken() {
```

Comment:
nit can we move this `getBrowserSafeClientToken()` function to the bottom of the file similar to our other examples?